diff --git a/src/main/ua-parser.mjs b/src/main/ua-parser.mjs
index c5b39b71dbc934d46b532b3bbde9fd365af19f6b..bd50dd9b4e46c57ec093f56fe42a62570122de80 100644
--- a/src/main/ua-parser.mjs
+++ b/src/main/ua-parser.mjs
@@ -1275,7 +1275,7 @@
                 this.set(MAJOR, majorize(this.get(VERSION)));
                 break;
             case OS:
-                if (this.get(NAME) == 'iOS' && this.get(VERSION) == '18.6') {
+                if (this.get(NAME) == 'iOS' && (+this.get(VERSION) >= 18.6)) {
                     // Based on the assumption that iOS version is tightly coupled with Safari version
                     var realVersion = /\) Version\/([\d\.]+)/.exec(this.ua); // Get Safari version
                     if (realVersion && parseInt(realVersion[1].substring(0,2), 10) >= 26) {
